sdk/communication/azure-communication-chat/src/samples/java/com/azure/communication/chat/ReadmeSamples.java

```diff
@@ -40,20 +37,14 @@
     public ChatClient createChatClient() {
         String endpoint = "https://<RESOURCE_NAME>.communcationservices.azure.com";
 
-        // Create an HttpClient builder of your choice and customize it
-        // Use com.azure.core.http.netty.NettyAsyncHttpClientBuilder if that suits your needs
-        NettyAsyncHttpClientBuilder httpClientBuilder = new NettyAsyncHttpClientBuilder();
-        HttpClient httpClient = httpClientBuilder.build();
-
         // Your user access token retrieved from your trusted service
         String token = "SECRET";
         CommunicationTokenCredential credential = new CommunicationTokenCredential(token);
 
         // Initialize the chat client
         final ChatClientBuilder builder = new ChatClientBuilder();
         builder.endpoint(endpoint)
-            .credential(credential)
-            .httpClient(httpClient);
+            .credential(credential);
         ChatClient chatClient = builder.buildClient();
```

Comment:
This line will throw

Reply:
This is now fixed so it will not throw anymore